pos_default_quantity/views/product_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
```

Comment:
you can remove data if noupdate is not set

Reply:
shame on me (bis)